Infrastructure-Scripts/Remove-AzSqlIpException.ps1

```diff
@@ -0,0 +1,51 @@
+<#
+    .SYNOPSIS
+    Remove the whitelisted IP from the firewall rule name
+    .DESCRIPTION
+    Remove the whitelisted IP from the firewall rule name
+    .PARAMETER Name
+    The name of the firewall rule
+    .PARAMETER ServerName
+    Name of the Sql server
+    .PARAMETER WhatsMyIpUrl
+    The url value, "https://ifconfig.me/ip"  that needs to get passed in to identify the IP Address
+    .PARAMETER  ResourceGroupName
```

Comment:
As mentioned in https://github.com/SkillsFundingAgency/das-platform-automation/pull/99#issuecomment-977230727, this parameter wouldn't be needed if a ResourceNamePattern parameter was used.